### ISSUE TITLE
Linux

### ISSUE THREAD

**Issue by PowaBanga** [smartties/Blip-Blop-for-Android#1]
Hi,
Do you think that you can make it work on linux ? ubuntu, fedora, opensuse etc...

**Comment by smartties** [comment]
Yes I could. Unfortunately, due to my studies and my job, I don't  have any free time to work on other project.
